src/Core/Migration/MigrationHandler.php

```diff
@@ -229,9 +237,41 @@ protected function buildMigrationQueries()
         }
 
         $oDirectory = new RecursiveDirectoryIterator($this->migrationQueriesDir);
-        $oFlattened = new RecursiveIteratorIterator($oDirectory);
+        $oIterators = new AppendIterator();
+        $oIterators->append(new RecursiveIteratorIterator($oDirectory));
+
+        // Include module migration files
+        if (!self::$skipModuleMigration) {
+            $oConfig = Registry::getConfig();
+
+            if (!class_exists(ModuleList::class)) {
+                throw new MigrationException('ERROR: Oxid ModuleList class can not be loaded,
+                please try to run vendor/bin/oe-eshop-unified_namespace_generator');
```

Comment:
Please remove the newline in the message. You can use two strings on two lines and concatenate them like so:
```php
throw new MigrationException('ERROR: Oxid ModuleList class can not be loaded, '
. 'please try to run vendor/bin/oe-eshop-unified_namespace_generator');
```

Reply:
Done.